recipes/vsclust/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "VSClust" %}
+{% set version = "0.86" %}
+
+package:
+  name: vsclust
+  version: {{ version }}
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+source:
+  url: https://bitbucket.org/veitveit/vsclust/get/release-0.86.tar.gz
+  sha256: d94808cc3d9bef2741fc9b597aa25e971db8f7422a6c30166b59a388c8714510
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+  host:
+    - r-base 3.6.3
+    - r-matrixstats >=0.56
+    - bioconductor-mfuzz >=2.46.0
+    - bioconductor-qvalue >=2.16.0
+    - bioconductor-limma
+  run:
+    - r-base
+    - r-matrixstats >=0.56.0
+    - bioconductor-mfuzz >=2.46.0
+    - bioconductor-qvalue >=2.16.0
+    - bioconductor-limma
+    - bioconductor-clusterprofiler 
+    - bioconductor-RDAVIDWebService
+    - r-readxl >=1.3.1
+    - r-shiny >=1.4.0
+    - r-yaml >=2.2.0
+    - r-shinyjs >=1.1
+    - r-shinythemes >=1.1.2
+    - r-data.table >=1.12.8
+    
+test:
+  commands:
+     - cp $PREFIX/bin/ProtExample.csv . ; runVSClust.R $PREFIX/bin/vsclust.yml
+
+about:
+  home: https://bitbucket.org/veitveit/vsclust/src/master/
+  license: 'GPL (>=2)'
+  license_family: GPL
+  license_file: LICENSE
+  summary: Interactive tool for statistical testing, data browsing and interactive visualization of quantitative omics data
+  description: |
+    VSClust is a web service (shiny app) and command-line tool for statistical testing, clustering and interactive visualization of quantitative omics data. Its variance-sensitive clustering algorithm improves identification of co-regulated features in noisy data with replicates
```

Comment:
This could be wrapped at 72 chars widths, so it can be read in github a little better.

Reply:
@epruesse 
Thanks for the detailed requests. 
Now the bin folder only contains the binaries and I also solved the other issues.